sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -29,6 +31,26 @@ export type AuthorizationRule = {
     secondaryKey?: string;
 };
 
+// @public
+export interface Closeable {
```

Comment:
Unused interface?

Reply:
@bterlson did have a case where `subscribe`, for instance, could return something that was itself closeable in order to end the subscription. 

Now that you're really just interacting with a single receiver it's probably not needed. Will remove.